src/cli.spec.js

```diff
@@ -9,11 +9,39 @@ const { setUp, tearDown } = require("./test-helpers.js");
 chai.use(require("chai-as-promised"));
 
 describe("CLI", function () {
+  // Moch the catalog validation schema
+  beforeEach(() => {
```

Comment:
Not specifically related to this PR. I should have already done it, but there is a lot of network IO going on that we need to mock. Lets add `nock.disableNetConnect()` to the start of `beforeEach()` too. This will stop nock from letting through any request to the network so if we forget to mock a request under test it will fail.

Reply:
I did not know this feature, I add it in 7b3428af9bfd65407dfceebcb7e6bd12de6de600